app/scripts/directives/util.js

```diff
@@ -149,6 +149,22 @@ angular.module('openshiftConsole')
       }
     };
   })
+  .directive('setHomePage', function($uibModal) {
+    return {
+      restrict: 'E',
+      replace: true,
```

Comment:
`replace` is deprecated. We should avoid it.

Reply:
The only problem is that without the `replace` the HTML is:
```
<dropdown-menu>
  <li>
       <set-home-page>   <--- not 'replaced'
           <a href...>
```
Which breaks the CSS dropdown selectors, which many have:
`.dropdown-menu > li > a`

So 'Set Home Page' in the user dropdown menu isn't styled like the other menu options.

Reply:
Not sure we want to override all the `.dropdown-menu > li > a` selectors

Reply:
The directive is simple enough I'd almost rather not have it be a directive. We could have a helper in a service to open the modal. Then all it is is a link with a one line click handler.

Reply:
I run into the same issue below where userDropdown.js is an extension where ng-click doesn't work, so I have to defer to using a directive.   The other `<a href=...`'s in the user and help dropdowns actually redirect to a URLs, but I don't think we want to do that for a modal.

Reply:
Its a little too bad that the selector is `.dropdown-menu > li > a` instead of `.dropdown-menu  .link > a` or something that would provide a little more flexibility...